src/loaders/lottie/tvgLottieModel.h

```diff
@@ -124,6 +124,7 @@ struct LottieObject
     }
 
     char* name = nullptr;
+    char* sid = nullptr;
```

Comment:
Could you please double check this?
```
Slot {
   char* sid;
   type;
   Array<LottieObject*> objs;   //assuming multiple properties uses the same sid.
};
```
I think LottieObject doesn't need to have sid member.